addons/project/static/src/components/project_control_panel/project_control_panel.xml

```diff
@@ -27,7 +27,7 @@
 
     <t t-name="project.Breadcrumbs.Small" t-inherit="web.Breadcrumbs.Small" t-inherit-mode="primary" owl="1">
         <xpath expr="//ol" position="inside">
-            <t t-call="project.ProjectControlPanelContent"/>
+            <div t-call="project.ProjectControlPanelContent"/>
```

Comment:
Thanks for your work, however, we cannot do that in stable because a custom code could do an xpath and that xpath could crash if it no longer reaches the element. 

Can you retarget your fix to master? 

Thanks in advance! 

cc @auon-odoo

Reply:
> Thanks for your work, however, we cannot do that in stable because a custom code could do an xpath and that xpath could crash if it no longer reaches the element.
> 
> Can you retarget your fix to master?
> 
> Thanks in advance!
> 
> cc @auon-odoo

In my opinion, scss is the best option to resolve the issue instead of XML.

Reply:
Hello @xavierbol How about fixing a minimum width for the set status button so the button could be displayed perfectly(only for small devices). Also the bug only occurs in 16.0 from saas-16.3 onwards every element is in a separate div already for breadcrumb.
cc: @auon-odoo